go/rpc/encrypted_client.go

```diff
@@ -251,18 +257,19 @@ func (c *EncRPCClient) encryptParamBytes(params []byte) ([]byte, error) {
 	return encryptedParams, nil
 }
 
-func (c *EncRPCClient) decryptResponse(resultBlob interface{}) ([]byte, error) {
+func (c *EncRPCClient) decryptHexString(resultBlob interface{}) ([]byte, error) {
```

Comment:
Had to break this out, as the logs don't come back as hex strings, unlike all other responses to decrypt.